src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -473,7 +483,16 @@ public void postBrapiData(Map<Integer, PendingImport> mappedBrAPIImport, Program
                 String dbid = createdTrial.getTrialDbId();
                 listedTrial.setTrialDbId( dbid );
             }
-            brAPILocationDAO.createBrAPILocation(newLocations, program.getId(), upload);
+
+            List<BrAPILocation> createdLocations = new ArrayList<>(brAPILocationDAO.createBrAPILocation(newLocations, program.getId(), upload));
```

Comment:
Not a suggestion for this card, but it may be worth having a card to refactor this logic to create a location via the `ProgramLocationService`, that way the location gets created in both the backing BrAPI server and in the BI DB

Reply:
Done